contracts/src/v0.8/shared/test/enumerable/EnumerableMapAddresses.t.sol

```diff
@@ -161,16 +218,34 @@ contract EnumerableMapAddresses_get is EnumerableMapAddressesTest {
     assertTrue(s_addressToBytes32Map.contains(address(this)));
     assertTrue(s_addressToBytes32Map.get(address(this)) == MOCK_BYTES32_VALUE);
   }
+
+  function testBytesGetSuccess() public {
+    assertTrue(!s_addressToBytesMap.contains(address(this)));
+    assertTrue(s_addressToBytesMap.set(address(this), MOCK_BYTES_VALUE));
+    assertTrue(s_addressToBytesMap.contains(address(this)));
+    assertTrue(keccak256(s_addressToBytesMap.get(address(this))) == keccak256(MOCK_BYTES_VALUE));
+  }
 }
 
 contract EnumerableMapAddresses_get_errorMessage is EnumerableMapAddressesTest {
   using EnumerableMapAddresses for EnumerableMapAddresses.AddressToBytes32Map;
   using EnumerableMapAddresses for EnumerableMapAddresses.AddressToAddressMap;
+  using EnumerableMapAddresses for EnumerableMapAddresses.AddressToBytesMap;
 
   function testGetErrorMessageSuccess() public {
     assertTrue(!s_addressToAddressMap.contains(address(this)));
     assertTrue(s_addressToAddressMap.set(address(this), address(this)));
     assertTrue(s_addressToAddressMap.contains(address(this)));
     assertTrue(s_addressToAddressMap.get(address(this), "EnumerableMapAddresses: nonexistent key") == address(this));
   }
+
+  function testBytesGetErrorMessageSuccess() public {
+    assertTrue(!s_addressToBytesMap.contains(address(this)));
+    assertTrue(s_addressToBytesMap.set(address(this), MOCK_BYTES_VALUE));
+    assertTrue(s_addressToBytesMap.contains(address(this)));
+    assertTrue(
```

Comment:
suggestion: same here - you can use `assertEq`

Reply:
updated